back/engines/commercial/analytics/lib/query.rb

```diff
@@ -39,12 +40,13 @@ def model
 
     def all_dimensions
       @all_dimensions ||= model
-        .reflect_on_all_associations(:belongs_to)
+        .reflect_on_all_associations()
         .to_h do |assoc|
           [
             assoc.name.to_s,
             {
-              columns: "Analytics::#{assoc.options[:class_name]}".constantize.new.attributes.keys,
+              # Note: Changed this to remove 'Analytics::'
+              columns: assoc.options[:class_name].constantize.new.attributes.keys,
```

Comment:
There might be a way to get the list of attributes without instantiating the class.